pkg/make_deb.py

```diff
@@ -24,6 +24,11 @@
 import textwrap
 import time
 
+if sys.version_info < (3, 7):
```

Comment:
version_info is a 5 tuple. The compare should be on parts 0 and 1 if we need it.

And why the 3.7 test? collections.OrderedDict is available since 3.1